Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py

```diff
@@ -125,7 +129,14 @@ def fetch_indicators_command(client: Client, indicator_type: str, feedTags: list
                 if not current_indicator_type:
                     continue
 
-                indicator = {'value': indicator_value, 'type': current_indicator_type, 'fields': {'tags': feedTags}}
+                indicator = {
+                    'value': indicator_value,
+                    'type': current_indicator_type,
+                    'fields': {
+                        'tags': feedTags,
+                        'trafficlightprotocol': client.tlp_color
```

Comment:
By setting it here, wouldn't the value be overridden by a tlp value that came in from the indicator feed source (assuming a case in which that data was provided as part of the feed)? I thought we said that the value set during instance configuration should take precedence. If that is the case, you would need to conditionally set this field after the mapping bit on lines 146-149.